### PR TITLE
Remove ed in `AccountBasic`

### PR DESCRIPTION
**Pull request by AsceticBear**

### CODE REVIEW

File: frame/dvm/ethereum/src/lib.rs [hunk 1]
```diff
@@ -386,13 +375,13 @@ pub mod pallet {
 	#[pallet::storage]
 	#[pallet::getter(fn get_ring_remaining_balances)]
 	pub(super) type RemainingRingBalance<T: Config> =
-		StorageMap<_, Blake2_128Concat, T::AccountId, RingBalance<T>, ValueQuery>;
+		StorageMap<_, Blake2_128Concat, T::AccountId, u128, ValueQuery>;
```
Comment (AsceticBear) [inline frame/dvm/ethereum/src/lib.rs:378]: Please be careful about this when reviewing. I think no migration need here because the type is the same.

Comment (AurevoirXavier) [reply frame/dvm/ethereum/src/lib.rs:378]: Recommend using `Currency::Balance`

Comment (AsceticBear) [reply frame/dvm/ethereum/src/lib.rs:378]: How to do that? I have deleted the `RingCurrency` and `KtonCurrency` in the `Config`.

Comment (AurevoirXavier) [reply frame/dvm/ethereum/src/lib.rs:378]: Never mind, I think no any other projects will use this pallet.

**Review by HackFisher — APPROVED** [review]

**Pull request merged**